be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/RollCallHandler.scala

```diff
@@ -66,7 +66,7 @@ case object RollCallHandler extends MessageHandler {
               //a closeRollCall is always sent in the lao's main channel so we are allowed to do this
               val socialChannel: String = generateSocialChannel(rpcMessage.getParamsChannel, head)
               val ask: Future[GraphMessage] = (dbActor ? DbActor.CreateChannel(Channel(socialChannel), ObjectType.CHIRP)).map {
-                case DbActorWriteAck() => createAttendeeChannels(tail, rpcMessage)
+                case DbActorAck() => createAttendeeChannels(tail, rpcMessage)
```

Comment:
I suppose that this case was not handled in the test suite, Would you please cover it ?

Reply:
(starting to be done in PR#701)